test/typescript/hello-world/greeter.service.ts

```diff
@@ -2,88 +2,96 @@
 
 import { Service } from "../../../";
 import { Context } from "../../../";
-import { ActionHooks, ServiceHooks, ServiceHooksAfter, ServiceSchema } from "../../../";
```

Comment:
Why you changed the test service file schema?

Reply:
I updated the schema to add the missing hooks now.